sql/src/main/java/io/crate/execution/engine/collect/files/FileReadingIterator.java

```diff
@@ -187,6 +202,18 @@ public boolean moveNext() {
         return false;
     }
 
+    private boolean isInputCsv() {
+        return (inputFormat == WriterProjection.InputFormat.CSV) || currentUri.toString().endsWith(".csv");
+    }
+
+    private void processAsJsonLine(String line) {
+        lineContext.rawSource(line.getBytes(StandardCharsets.UTF_8));
+    }
+
+    private void processAsCSV(String header, String line) throws IOException {
+        lineContext.rawSourceFromCSV(header.getBytes(StandardCharsets.UTF_8), line.getBytes(StandardCharsets.UTF_8));
```

Comment:
These two methods should be handled by the LineContext implementation. The interfacing method should remain `rawSource` or maybe just `source`.

Reply:
I've removed this method (`rawSourceFromCSV`) since it's no longer necessary